plugins/mqtt/package.json

```diff
@@ -29,17 +29,18 @@
       ]
    },
    "dependencies": {
-      "@types/node": "^16.6.1",
       "aedes": "^0.46.1",
-      "axios": "^0.23.0",
+      "axios": "^1.6.7",
```

Comment:
this version of axios adds a 4MB dependency on mime db. it's why I haven't upgraded. I'm actually migrating off axios and using fetch directly since that is in node 18.

Reply:
reverted